src/hipscat_import/map_reduce.py

```diff
@@ -0,0 +1,138 @@
+"""Import a set of non-hipscat files using dask for parallelization"""
+
+import os
+
+import dask.dataframe as dd
```

Comment:
why do we have two separate files name `dask_map_reduce` and `map_reduce` that both use dask? I feel like that might cause some complaints if a user is trying to avoid using dask 😅 maybe the first one should be `parallel_map_reduce` or something like that?

Reply:
Fair point. This was an attempt to keep the dask-y pipeline worker setup distinct from the interesting logic, mostly for testing and cleanliness. Since the whole pipeline relies on dask, the name of the file is kinda silly.